### PR TITLE
chore(windows): move to vc++ 2019

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

File: windows/src/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ For local development you do not need to perform a release build so these are op
 
 ## Building Keyman for Windows and Keyman Developer
 
-1. Start 'x64_x86 Cross Tools Command Prompt for VS 2017'.
+1. Start 'Developer Command Prompt for VS 2019'.
```
Comment (rc-swag) [inline windows/src/README.md:32]: Why not "x64_x86 Cross Tools Command Prompt for VS 2019"? I don't know the difference but I can see when I run the "x64_x86" prompt the file vcvarsall.bat is run.

Comment (mcdurdin) [reply windows/src/README.md:32]: We don't really need the Cross Tools Command Prompt. That's for running x64-specific tooling on an x86 system, I think. The normal "Developer Command Prompt" works fine.

**Review by rc-swag — APPROVED** [review]
lgtm

**Comment by darcywong00** [comment]
ack, the Test-14.0 Windows build failed :(

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.53-alpha](https://keyman.com/downloads/releases/alpha/15.0.53)

**Pull request merged**